Modules/Synchronization/test/Synchronization.Application.Tests/Tests/DomainEvents/ExternalEventCreatedDomainEventTests.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Backbone.DevelopmentKit.Identity.ValueObjects;
+using Backbone.Modules.Synchronization.Domain.DomainEvents.Outgoing;
+using Backbone.Modules.Synchronization.Domain.Entities.Sync;
+using FluentAssertions;
+using Xunit;
+
+namespace Backbone.Modules.Synchronization.Application.Tests.Tests.DomainEvents;
+
+public class ExternalEventCreatedDomainEventTests
+{
+    [Fact]
+    public void Raises_a_domain_event_on_initialization()
+    {
+        var identity = TestDataGenerator.CreateRandomIdentityAddress();
+        var externalEvent = new ExternalEvent(ExternalEventType.IdentityDeletionProcessStatusChanged, IdentityAddress.Parse(identity), 1, new { });
```

Comment:
Why do you parse an IdentityAddress to an IdentityAddress?

```suggestion
        var externalEvent = new ExternalEvent(ExternalEventType.IdentityDeletionProcessStatusChanged, identity, 1, new { });
```